src/Microsoft.AspNet.Identity.EntityFramework/IdentityUser.cs

```diff
@@ -87,5 +88,20 @@ public IdentityUser(string userName) : this()
         ///     Used to record failures for the purposes of lockout
         /// </summary>
         public virtual int AccessFailedCount { get; set; }
+
```

Comment:
We actually don't have proxies in EF7 so far. @ajcvickers what do you think the best possible pattern would be for these collection navprops?